### ISSUE TITLE
Suggest Categories

### ISSUE THREAD

**Issue by SachaG** [SachaG/SidebarFeedback#1]
Suggest new categories for Sidebar here.

**Comment by a-foutoyet** [comment]
Website, Guidelines, Redesign, Tools

**Comment by a-foutoyet** [comment]
Guidelines

**Comment by SachaG** [comment]
@a-foutoyet what would go in the "website" and "guidelines" categories? And there's already "resources" for tools.

**Comment by SachaG** [comment]
And redesign already exists :)

**Comment by a-foutoyet** [comment]
Because for searching, inspiration could be applied to wider things than only websites. So I thought it could be a good idea to add a website tag to only search for websites. And concerning guidelines, I saw there are some links refering to guidelines like (https://css-tricks.com/style-guide-driven-development-atomic-docs/ or https://developer.apple.com/ios/human-interface-guidelines/ etc.). I didn't see the "Redesign" tag on the right, that's why I thought it doesn't exist but I got it know, sorry ! For the tools >> ressources tags ok, I have to learn what link goes under what tag !

**Comment by SachaG** [comment]
Well there's already the Style Guides tag, I think that's pretty close. As for websites, pretty much everything linked is a website I think? I don't really see how it would come in handy?

**Comment by a-foutoyet** [comment]
No prob, you're right, I'll use the style guide tag now.  Concerning inspiration, it was just an idea but if everything behind is a website then that's good !

**Comment by robwco** [comment]
Business? :)

**Comment by SachaG** [comment]
@robwco actually I would prefer to keep Sidebar away from more business-related content. There's a ton of great posts out there about the business of design, but I don't really want Sidebar to be about that.

**Comment by TedGoas** [comment]
Hey, where did **Article** and **Tutorial** go?

**Comment by TedGoas** [comment]
and **Tutorial**?

**Comment by SachaG** [comment]
`Article` seems a bit vague to me, it could apply to basically anything. I do have `Essay` though. But `Tutorial` is a good idea :)

**Comment by TedGoas** [comment]
Yea, the I thought he vagueness of **Articles** made sense. Anything from a feature article on A List Apart to a blog post on a personal site. I guess **Essays** is the same thing?

**Comment by SachaG** [comment]
Yes **Essays** is kind of a catch-all already, so I don't think we also need **Articles**.

**Comment by a-foutoyet** [comment]
I've searched for article too. I thought essay was more specific too really big articles but it's good to know I could use it. There is also learning we could use.

**Comment by loriskumo** [comment]
How about: visual design, design, graphic design?
Or user testing? :-)

**Comment by zhouzi** [comment]
It could be nice to have access to the full list. When submitting a link, the user may be unsure to what tag the link relates best so having all the options could help.

**Comment by beckiiadel** [comment]
psychology!

**Comment by Richa2605** [comment]
We would be glad to see these tags also.
1) Productivity 
2) Startup
3) Early users 
4) Web development

**Comment by ziyafenn** [comment]
Chatbot, bot

**Comment by imcatnoone** [comment]
Games

**Comment by tpalmer75** [comment]
Workflow / Tools

**Comment by robv** [comment]
Reviews (Book, Software, etc)

**Comment by JohnMooreWilliams** [comment]
Content Strategy, Voice and Tone

**Comment by caioabraga** [comment]
Automation :)

**Comment by roomfive** [comment]
product design, system design

**Comment by CrisLindner** [comment]
complex system design, future design, design fractal thinking

**Comment by prashantsani** [comment]
Technologies : 
SVG, JavaScript, WebGL, Greensock, GSAP, ScrollMagic, VueJS, ReactJS

Type:
Case Study, Tutorial, Reviews (books/tech) (adding in @robv  ) , Inspiration, Work-life Balance, Website Performance

**Comment by bhatiak** [comment]
Research

**Comment by caioabraga** [comment]
Diversity, Career, Ethics, Team

**Comment by IvdLooij** [comment]
HR, Recruitment, Team

**Comment by friendlyswiss** [comment]
Product Management

**Comment by MrMage** [comment]
How about a Mac category?

**Comment by BuddhaSource** [comment]
Making, Building?

**Comment by vlafiser** [comment]
React, React Native

**Comment by sbvs** [comment]
Diversity, inclusion

**Comment by daninfpj** [comment]
Process

**Comment by efekurnaz** [comment]
Security!

**Comment by roughprada** [comment]
Privacy

**Comment by firatdemirel** [comment]
Domains or Domain Names
Big Data

**Comment by eldritchterror** [comment]
Games UX / Video games UX 
:)

**Comment by AnandChowdhary** [comment]
Open Source

**Comment by joaoramos** [comment]
I'd suggest
* Product Design
* Design Methods
* Collaboration
* Design Strategy
* Systems Thinking

**Comment by jimmymarks** [comment]
- Empathy
- Application design
- User research

**Comment by ghost** [comment]
Ideation

**Comment by kingsidharth** [comment]
Design Management

**Comment by MvRemmerden** [comment]
Conferences

**Comment by dmshvetsov** [comment]
Programming

**Comment by fabric-8** [comment]
Design Process

**Comment by karenkliu** [comment]
Ethics

**Comment by roppem9** [comment]
voice design or conversational design?
